app/(service)/layout.tsx

```diff
@@ -1,20 +1,14 @@
-import { Inter } from 'next/font/google';
-
 import Gnb from '@/components/gnb/gnb';
 
-const inter = Inter({ subsets: ['latin'] });
-
 export default function RootLayout({
   children,
 }: Readonly<{
   children: React.ReactNode;
```

Comment:
P3: 이렇게 react에서 import해서 사용하는 게 통일성을 위해 좋을 것 같습니다 :)

```suggestion
  children: ReactNode;
```